### PR TITLE
Upgraded lang-detection to Julia 1.0 (With the correct branch this time)

### PR DESCRIPTION
**Pull request by NTimmons**
Creating a new PR due to changing branch, and I think this will be easier than fighting Github.

model.jl 
- Renamed the normalize_string function to normalize from the Unicode library
- Added 'using Random' for the shuffle function
- Added 'using Statistics' for the mean function

scrape.jl
- Changed matchall to eachmatch from the Cascadia library. Which appears to behave the same and is consistent with the usage guide on the github page for that library (https://github.com/Algocircle/Cascadia.jl)
- moved from sel"some regex" to Selector("some regex") in the match call so that a new using can easily find the Selector method. (I had trouble finding the documentation for the sel prefix)
- IOBuffer has changed. Switched innerText to work on a string instead. Might be less efficient but task is trivial.

### CODE REVIEW

**Comment by dhairyagandhi96** [comment]
This is great, thanks!

**Comment by dhairyagandhi96** [comment]
Thoughts on moving training over to the GPU?

**Comment by dhairyagandhi96** [comment]
Bump!

It would be nice to get this merged soon. Thanks for all the help!

**Comment by NTimmons** [comment]
Apologies for the delay. I added the gpu parts here 2c8c1e2abdf2b5fcc04bf2c92f946432593ba5ce

**Comment by dhairyagandhi96** [comment]
The CUDA manifest should be separate (in a coda folder), so we have both CPU and GPU versions.

File: text/lang-detection/model.jl [hunk 1]
```diff
@@ -42,8 +46,7 @@ loss(x, y) = crossentropy(model(x), y)
 
 testloss() = mean(loss(t...) for t in test)
 
-opt = ADAM()
-ps = params(scanner, encoder)
+opt = ADAM(params(scanner, encoder))
```
[the remark this answers is not in the document]
Comment (NTimmons) [reply text/lang-detection/model.jl]: Oh! Apologies, I have been running an old version for another project. I will fix this.

**Comment by dhairyagandhi96** [comment]
Bump @NTimmons

**Comment by dhairyagandhi96** [comment]
Closing in favour of #122

**Pull request closed without merging**